lib/libzfs/os/freebsd/libzfs_compat.c

```diff
@@ -39,12 +39,6 @@
 #define	ZFS_KMOD	"openzfs"
 #endif
 
-void
-libzfs_set_pipe_max(int infd)
```

Comment:
you should should probably remove the declaration from `libzfs_impl.h` if you're removing all definitions

Reply:
Good idea. Done.